datascience_eda/datascience_eda.py

```diff
@@ -90,4 +95,73 @@ def initial_clustering(
 
     # add all plots to result dictionary
 
-    return result
\ No newline at end of file
+    return result
+
+
+def eda_text_columns(df, text_col = None, params = dict()):
+    """Perfroms EDA of text features.
+    - prints the summary statistics of character length
+    - plots the distribution of character length
+    - prints the summary statistics of word count
+    - plots the distribution of word count
+    - plots the word cloud
+    - plots bar chart of top n stopwords
+    - plots bar chart of top n words other than stopwords 
+    - plots bar chart of top n bigrams
+    - plots the distribution of polarity and subjectivity scores
+    - plots bar charts of sentiments, name entities and part of speech tags
+
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        the dataset (X)
+    text_col : str
+        name of text column
+    params : dict
+        a dictionary of parameters
+
+    Returns
+    -------
+    None
```

Comment:
Tiff said we should always return something, probably return a list of plots?

Reply:
Addressed this in the new [PR](https://github.com/UBC-MDS/datascience_eda/pull/17)